Gemfile

```diff
@@ -1,7 +1,9 @@
 source 'https://rubygems.org'
-ruby '2.1.2'
+ruby '2.2.3'
 
 gem 'rails', '4.1.2'
+gem 'puma', '~> 2.13.4'
```

Comment:
I can fix in my testing

On Fri, Aug 21, 2015 at 4:43 PM, Jon McCartie <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/rubycentral/cfp-app/pull/51#discussion_r37685121>:
>
> >
> >  gem 'rails', '4.1.2'
> > +gem 'puma', '~> 2.13.4'
>
> Agreed. I'll make another PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rubycentral/cfp-app/pull/51/files#r37685121>.
>